complaint_search/tests/test_es_interface.py

```diff
@@ -36,11 +37,307 @@ def test_search_no_param__valid(self, mock_rget, mock_search):
                 "fragment_size": 500
             },
             "sort": [{"_score": {"order": "desc"}}],
-            "post_filter": {"and": {"filters": []}}
+            "post_filter": {"and": {"filters": []}},
+            "aggs": {
```

Comment:
I was thinking, how in the world is the test file 9K lines?  I see now it is due to lots of embedded JSON to validate the responses.

It's fine, but you could always store them as static files in the same directory and keep the test file focused on the test, not the expected data.  Here's my version of that pattern

```python
    # -------------------------------------------------------------------------
    # Helper Methods
    # -------------------------------------------------------------------------

    def to_absolute(self, fileName):
        import os.path
        # where is this module?
        thisDir = os.path.dirname(__file__)
        return os.path.join(thisDir, fileName)

    def load(self, shortName):
        import json
        fileName = self.to_absolute(shortName + '.json')
        with open(fileName, 'r') as f:
            return json.load(f)

    # -------------------------------------------------------------------------
    # Test Methods
    # -------------------------------------------------------------------------

    def test_search_no_param_valid(self):
        expected = self.load('search_no_param_valid')

        actual = ...

        self.assertEqual(actual, expected)
```